src/widgets/allergies/allergies-overview.component.tsx

```diff
@@ -77,18 +77,25 @@ const AllergiesOverview: React.FC<AllergiesOverviewProps> = () => {
       return (
         <div>
           <div className={styles.allergiesHeader}>
-            <h4>{headerTitle}</h4>
+            <h4 className={`${styles.productiveHeading03} ${styles.text02}`}>
+              {headerTitle}
+            </h4>
             <Button
               kind="ghost"
               renderIcon={Add16}
               iconDescription="Add allergies"
```

Comment:
Sorry - have not seen that we still have some icon descriptions in there.

Would you mind taking care of all of them? Much appreciated!

Reply:
Those don't get displayed in the UI. They're used as aria labels.

Reply:
Perfect - thanks for clarifying!